base_station_nodes: &id_base_stations
  - WayPoint0
wait_nodes: *id_base_stations
local_storage_node: WayPoint1
charging_station_node: WayPoint2
robot_ids: 
  - robot00
max_task_priorities: 
  - 255
picker_ids:
  - picker01
  - picker02
  - picker03
  - picker04
  - picker05
  - picker06
  - picker07
  - picker08
  - picker09
  - picker10
virtual_picker_ids:
  - picker101
  - picker102
  - picker103
  - picker104
  - picker105
  - picker106
  - picker107
  - picker108
  - picker109
  - picker110
